polars/01_why_polars.py

```diff
@@ -0,0 +1,299 @@
+# /// script
+# requires-python = ">=3.12"
+# dependencies = [
+#     "marimo",
+#     "pandas==2.2.3",
+#     "polars==1.22.0",
+# ]
+# ///
+
+import marimo
+
+__generated_with = "0.11.0"
+app = marimo.App(width="medium")
+
+
+@app.cell
+def _():
+    import marimo as mo
+    return (mo,)
+
+
+@app.cell
+def _(mo):
+    mo.md(
+        """
+        # An introduction to Polars
+
+        This notebook provides a birds-eye overview of [Polars](https://pola.rs/), a fast and user-friendly data manipulation library for Python, and compares it to alternatives like Pandas and PySpark.
+        
+        Like Pandas and PySpark, the central data structure in Polars is **the DataFrame**, a tabular data structure consisting of named columns. For example, the next cell constructs a DataFrame that records the gender, age, and height in centimeters for a number of individuals. 
+        
+        <INSERT CODE CELL>
```

Comment:
The <INSERT CODE CELL> suggestion was to split this markdown into two cells, and insert a block of Python in between that creates a DataFrame (for example, the gender, age, and height dataframe) which is reused in subsequent cells. Perhaps

```python
    import polars as pl

    df_pl = pl.DataFrame(
        { 
            "gender": ["Male", "Female", "Male", "Female", "Male", "Female", 
                       "Male", "Female", "Male", "Female"],
            "age": [13, 15, 17, 19, 21, 23, 25, 27, 29, 31],
            "height_cm": [150.0, 170.0, 146.5, 142.0, 155.0, 165.0, 170.8, 130.0, 132.5, 162.0]
        }
    )
    df_pl
```